### PR TITLE
pre-alpha - Templates meta information popup implementaion

### PR DESCRIPTION
**Pull request by Karthigaiselvan**
## Release Checklist

- All steps below must be completed prior to merging.

## Changes Made

- Please note all changes made to API, design, functionality, etc.

### Required

- [ ] PR title should follow correct format.
  - MAJOR Release is used when there are breaking changes involved
  - MINOR Release is used when enhancements are introduced
  - PATCH Release is used for defects, security, tech chores, etc.
  - pre-alpha is used for testing releases
  - Note:
    - Based on the commit messages, increment the version from the latest release.
    - Push the bumped npm version in package.json back into the repo.
    - Push a tag for the new version back into the repo.
    - [For more details on versioning](https://github.com/phips28/gh-action-bump-version/blob/master/README.md) 
- [ ] Checklist of **changes made** added to PR description
- [ ] Related issue(s) linked to PR

 
## Reviewer Checklist

- [ ] Author checklist has been reviewed?
- [ ] All acceptance criteria has been met from linked issue
- [ ] Run code locally and verify all changes

## Related Issue(s)

Resolves #XXXX
Story-######
Defect-######

### CODE REVIEW

File: plugins/dai-release/src/components/MetaModalPopupComponent/MetaModalPopupComponent.tsx [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+  return (
+    <div>
+      <BootstrapDialog
+        onClose={onClose}
+        aria-labelledby="customized-dialog-title"
+        open={openModal}
+        className={!modalPopupData ? 'loading' : ''}
+      >
+        {!modalPopupData && <CircularProgress />}
+
```
Comment (sishwarya) [inline plugins/dai-release/src/components/MetaModalPopupComponent/MetaModalPopupComponent.tsx:147]: Can we have this modal popup as dynamic?

File: plugins/dai-release/src/components/MetaModalPopupComponent/MetaModalPopupComponent.tsx [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+  openModal: boolean;
+  onClose: () => void;
+  instance: string;
+  modalPopupInputId: string;
+  modalTitle: string;
+  modalPopupData: any;
+  setModalPopupData: (data: any) => void;
+};
+
+export function MetaModalPopupComponent({
```
Comment (sishwarya) [inline plugins/dai-release/src/components/MetaModalPopupComponent/MetaModalPopupComponent.tsx:21]: Rename has popupComponent

**Review by sishwarya — CHANGES_REQUESTED** [review]

File: plugins/dai-release/src/components/MetaModalPopupComponent/MetaModalPopupComponent.tsx (hunk 2 above)
Comment (Karthigaiselvan) [reply plugins/dai-release/src/components/MetaModalPopupComponent/MetaModalPopupComponent.tsx]: As per the discussion modified to "ModalComponent"

File: plugins/dai-release/src/api/DaiReleaseApiClient.ts [hunk 3]
```diff
@@ -30,7 +31,7 @@ export class DaiReleaseApiClient implements DaiReleaseApi {
 
   private async getToken() {
     const { token } = await this.identityApi.getCredentials();
-    return token;
+    return token; //  '6HqrCFFDme/3WcC0SFdpaunTu8g2WY+/'
```
[the remark this answers is not in the document]
Comment (Karthigaiselvan) [reply plugins/dai-release/src/api/DaiReleaseApiClient.ts]: done

**Review by sishwarya — APPROVED** [review]
LGTM

**Pull request merged**